tests/conftest.py

```diff
@@ -47,6 +49,84 @@ def client():
         yield client
 
 
+@pytest.fixture(scope="function")
+def list_policies_with_resource_path_patcher(test_data):
+    resource_paths = (
+        [test_data["resource_path"]] if isinstance(test_data, dict) else test_data
+    )
+    list_policies_mock = MagicMock()
+    future = asyncio.Future()
+    future.set_result(
+        get_list_policies_response(
+            arborist.get_auto_policy_id_for_resource_path(resource_paths[0]),
+            resource_paths,
+        )
+    )
+    list_policies_mock.return_value = future
+    policy_expand_patch = patch(
+        "requestor.routes.query.arborist.list_policies", list_policies_mock
+    )
+    policy_expand_patch.start()
+    yield
+    policy_expand_patch.stop()
+
+
+@pytest.fixture(scope="function")
+def list_policies_with_policy_id_patcher(test_data):
```

Comment:
i think `list_policies_with_resource_path_patcher` and `list_policies_with_policy_id_patcher` could be a single fixture, with `if "policy_id" not in test_data: call get_auto_policy_id_for_resource_path` and appropriate comments

and the data can be defined in the fixture instead of in `get_list_policies_response`